src/CustomElements/v1/CustomElements.js

```diff
@@ -61,27 +61,18 @@ var CustomElementDefinition;
     return reservedTagList.indexOf(name) !== -1;
   }
 
-  /**
-   * @constructor
-   * @property {Map<String, CustomElementDefinition>} _defintions
-   * @property {MutationObserver} _observer
-   * @property {MutationObserver} _attributeObserver
-   * @property {HTMLElement} _newInstance
-   * @property {string} _newTagName
-   * @property {boolean} polyfilled
-   */
-  function CustomElementsRegistry() {
-    this._definitions = new Map();
-    this._observer = this._observeRoot(document);
-    this._attributeObserver =
-        new MutationObserver(this._handleAttributeChange.bind(this));
-    this._newInstance = null;
-    this._newTagName = null;
-    this.polyfilled = true;
-  }
-
-  /** @lends {CustomElementsRegistry.prototype} */
-  CustomElementsRegistry.prototype = {
+  /** @export */
```

Comment:
Remove. `@export` doesn't do anything, without `--generate_exports`, and then it requires runtime support.